README.md

```diff
@@ -22,6 +22,36 @@ See [this](https://github.com/angular/angular/blob/master/README.md) for up-to-d
 * Edge (20+)
 * Safari (7+)
 
+## Dependencies
+* [Angular 2](https://angular.io) (tested with 2.0-rc.4)
+* [Bootstrap 4](https://v4-alpha.getbootstrap.com) (test with 4.0 alpha V2)
+
+## Installation
+As soon as you've got the above dependencies installed, install `ng-bootstrap` via:
+```
+npm install --save @ng-bootstrap/ng-bootstrap
+```
+Once Installed, directives may be imported in several different ways:
+For all the directives (if you simply intend on using everything):
+```
+import {NGB_DIRECTIVES} from '@ng-bootstrap/ng-bootstrap';
+```
+Or, for specific component directives (using Alert as an example):
+```
+import {NGB_ALERT_DIRECTIVES} from '@ng-bootstrap/ng-bootstrap';
+```
+Once imported, add your chosen directives on your component's `directives` array property:
+```
+directives: [
+  ...,
+  NGB_DIRECTIVES,
+  ...
+]
+```
+Certain directives need to be precompiled in order to be used. This is done via your component's `precompiled` array property. The following components require precompilation:
+* `NgbAlert`
+* `NgbPopoverWindow`
+* `NgbTooltipWindow`
```

Comment:
I don't think we need / want to enumerate those. Let's just tell people to:
* `import { NGB_PRECOMPILE } from '@ng-bootstrap/ng-bootstrap';`
* add `NGB_PRECOMPILE` to the `precompile` attribute of the _root_ (app) component

At the end of the say it should boils to sth like:

```javascript
import { Component } from '@angular/core';
import { NGB_DIRECTIVES, NGB_PRECOMPILE } from '@ng-bootstrap/ng-bootstrap';

@Component({
  selector: 'app',
  directives: [ NGB_DIRECTIVES ],
  precompile: [ NGB_PRECOMPILE ],
  templateUrl: './app.html',
})
export class App {
    ...
}
```